### DISCUSSION TITLE
Help needed on the JSON Schema Pydantic 2.0 docs?

### DISCUSSION THREAD

**Discussion by logankilpatrick** [pydantic/pydantic#6372] [Question] [answered]
Hey folks! Congrats on the 2.0 release. OpenAI uses JSON Schema for the new function calling capability in our ChatCompletions API so I would be happy to help update the JSON Schema docs to work with Pydantic 2.0. Can I get some guidance on how I should go about this? I imagine the best path would be to read the migration guide and see what needs to change (if anything?).

**Comment by samuelcolvin** [comment, marked as answer]
Thats great to hear, thanks so much for letting us know.

Any help with the documentation would be awesome. Hopefully contributing to documentation should be fairly self explanatory, but if you find anything confusing please let us know and  we'll try to fix it or add docs about it to the contribution guide.

Almost all code examples in the docs are run, tested and linted with `pytest tests/test_docs.py`, if you want to reformat the examples or insert the print statement output automatically run `pytest tests/test_docs.py --update-examples`.

Let us know if synchronous discussion would be easier, we can easily find a place to chat.

**Reply by adriangb** [reply]
@logankilpatrick just to clarify, are you talking about our docs or your docs? I think Samuel interpreted it to mean our docs, but I know you also have docs w/ JSON schemas [like these](https://platform.openai.com/docs/api-reference/edits/create) so maybe you meant your docs?

**Comment by Kludex** [comment]
What I remember now, specifically about that page, is that the "Fields section" should reference the fields.md page.

**Reply by samuelcolvin** [reply]
👍 yup, the whole thing with `Field` being documented in this page is very old and weird and should be removed.
